### PR TITLE
List and item entities

### PR DESCRIPTION
**Pull request by nouran77**

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
     - { resource: sonata.yml }
+    - { resource: "@TodoBundleListBundle/Resources/config/services.yml" }
```
Comment (ahmed-alnaqa) [inline app/config/config.yml]: No need for this file, you can remove it.

File: src/Todo/Bundle/ListBundle/Entity/Item.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string")
```
Comment (ahmed-alnaqa) [inline src/Todo/Bundle/ListBundle/Entity/Item.php]: I think it should be `text` not `string`

File: src/Todo/Bundle/ListBundle/Entity/Item.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    /**
+     * @param mixed $list
+     */
+    public function setList(ListItem $list)
+    {
+        $this->list = $list;
+    }
+
```
Comment (ahmed-alnaqa) [inline src/Todo/Bundle/ListBundle/Entity/Item.php]: Where're the other setters and getters of the entity fields?

File: src/Todo/Bundle/ListBundle/Entity/ListItem.php [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * List
+ *
+ * @ORM\Table(name="lists")
+ * @ORM\Entity(repositoryClass="Todo\Bundle\ListBundle\Repository\ListRepository")
+ */
+class ListItem
```
Comment (ahmed-alnaqa) [inline src/Todo/Bundle/ListBundle/Entity/ListItem.php]: `List` not `ListItem`

File: src/Todo/Bundle/ListBundle/Repository/ItemRepository.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?php
```
Comment (ahmed-alnaqa) [inline src/Todo/Bundle/ListBundle/Repository/ItemRepository.php]: If you're not using this repo, then remove it.

File: src/Todo/Bundle/ListBundle/Repository/ListRepository.php [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<?php
```
Comment (ahmed-alnaqa) [inline src/Todo/Bundle/ListBundle/Repository/ListRepository.php]: If you're not using this repo, then remove it.

File: src/Todo/Bundle/ListBundle/Resources/config/services.yml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+services:
```
Comment (ahmed-alnaqa) [inline src/Todo/Bundle/ListBundle/Resources/config/services.yml]: You don't need this file right now.

**Review by ahmed-alnaqa — CHANGES_REQUESTED** [review]
Make sure that your files don't have any additional empty lines :)

File: src/Todo/Bundle/ListBundle/Entity/ListItem.php (hunk 4 above)
Comment (nouran77) [reply src/Todo/Bundle/ListBundle/Entity/ListItem.php]: php storm is considered List as a reserved word

File: src/Todo/Bundle/ListBundle/Repository/ItemRepository.php (hunk 5 above)
Comment (nouran77) [reply src/Todo/Bundle/ListBundle/Repository/ItemRepository.php]: i will need this repo

Comment (ahmed-alnaqa) [reply src/Todo/Bundle/ListBundle/Repository/ItemRepository.php]: Then create it when you want to write any functions inside it, otherwise remove it :)

Comment (nouran77) [reply src/Todo/Bundle/ListBundle/Repository/ItemRepository.php]: okay

**Review by ahmed-alnaqa — APPROVED** [review]

**Pull request merged**